controllers/errorHandlingControllers.js

```diff
@@ -1,6 +1,10 @@
 exports.handlePSQL400Errors = (err, req, res, next) => {
   if (err.code === '22P02') {
     res.status(400).send({ msg: 'Invalid Path Request' });
+  } else if (err.code === '23502') {
+    res.status(400).send({ msg: 'Comment body empty!' });
+  } else if (err.code === '23503') {
```

Comment:
This code is a generic error that says the key you're looking for doesn't exist - "Author not found" will only work for one endpint.

So you should change the error msg to be more generalized - eg "Invalid Input"